README.md

```diff
@@ -56,28 +64,100 @@ func main() {
 }
 ```
 
+Note that when using an authenticated Client, all calls made by the client will
+include the specified OAuth token. Therefore, authenticated clients should
+almost never be shared between different users.
+
 See the [oauth2 docs][] for complete instructions on using that library.
 
 For API methods that require HTTP Basic Authentication, use the
 [`BasicAuthTransport`](https://godoc.org/github.com/google/go-github/github#BasicAuthTransport).
 
+### Rate Limiting ###
+
+GitHub imposes a rate limit on all API clients. Unauthenticated clients are
+limited to 60 requests per hour, while authenticated clients can make up to
+5,000 requests per hour. To receive the higher rate limit when making calls
+that are not issued on behalf of a user, use the
+`UnauthenticatedRateLimitedTransport`.
+
+The `Rate` method on a client returns the rate limit information based on the most
+recent API call. This is updated on every call, but may be out of date if it's
+been some time since the last API call and other clients have made subsequent
+requests since then. You can always call `RateLimits()` directly to get the most
+up-to-date rate limit data for the client.
+
+To detect an API rate limit error, you can check if its type is `*github.RateLimitError`:
+
+```go
+repos, _, err := client.Repositories.List("", nil)
+if _, ok := err.(*github.RateLimitError); ok {
+	log.Println("hit rate limit")
+}
+```
+
+Learn more about GitHub rate limiting at
+http://developer.github.com/v3/#rate-limiting.
+
+### Conditional Requests ###
+
+The GitHub API has good support for conditional requests which will help
+prevent you from burning through your rate limit, as well as help speed up your
+application. `go-github` does not handle conditional requests directly, but is
+instead designed to work with a caching `http.Transport`. We recommend using
+https://github.com/gregjones/httpcache, which can be used in conjunction with
+https://github.com/sourcegraph/apiproxy to provide additional flexibility and
```

Comment:
> `go-github` does not handle conditional requests directly, but is
> instead designed to work with a caching `http.Transport`. We recommend using
> https://github.com/gregjones/httpcache, which can be used to provide additional
> flexibility and control of caching rules.

I don't think that sounds right. It makes it sound like `gregjones/httpcache` is used to provide flexibility and control over caching rules, but in fact that package is what provides actual HTTP client-side caching. That sentence previously applied to `apicache`, which indeed did provide customization over http caching rules.

I think it should be more like this:

> `go-github` does not handle conditional requests directly, but is
> instead designed to work with a caching `http.Transport`. We recommend using
> https://github.com/gregjones/httpcache for that.